allocator/mimalloc-sys/mimalloc/include/mimalloc-new-delete.h

```diff
@@ -0,0 +1,66 @@
+/* ----------------------------------------------------------------------------
+Copyright (c) 2018-2020 Microsoft Research, Daan Leijen
+This is free software; you can redistribute it and/or modify it under the
+terms of the MIT license. A copy of the license can be found in the file
+"LICENSE" at the root of this distribution.
```

Comment:
I know this is Microsoft... but just for the sake of consistency, shall we copy in `LICENSE` into the parent folder as well?